app/src/main/java/com/insudev/euvictodo/buisnessLogic/UseCase.kt

```diff
@@ -0,0 +1,115 @@
+package com.insudev.euvictodo.buisnessLogic
+
+import com.insudev.euvictodo.models.Filters
+import com.insudev.euvictodo.models.ModelInterface
+import com.insudev.euvictodo.models.Sorting
+import com.insudev.euvictodo.models.TodoModel
+import com.insudev.euvictodo.mvi.TodoListChangeResult
+import io.reactivex.Observable
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+class UseCase {
+
+    private val retrofit = Retrofit.Builder()
```

Comment:
to powinno być w innym pliku, między innymi ze względu na to, że taka konfiguracja retrofita może być obszerniejsza ale ważniejesze jest tutaj single responsibility